.github/workflows/push.yml

```diff
@@ -232,10 +232,39 @@ jobs:
           -verbose -batch_size 10 \
           -beam_size 1 \
           -seed 1 \
-          -random_sampling_topk=-1 \
-          -random_sampling_temp=0.0001 \
+          -random_sampling_topk -1 \
+          -random_sampling_temp 0.0001 \
+          -ban_unk_token \
           -out /tmp/gen
         diff data/data_lm/gen-sampling-sol.txt /tmp/gen && rm /tmp/gen
+    - name: Test LM generation with random top-k/nucleus sampling
+      run: |
+        python translate.py -model onmt/tests/test_model_lm.pt \
+          -src data/data_lm/src-gen.txt \
+          -verbose -batch_size 10 \
+          -beam_size 1 \
+          -seed 1 \
+          -random_sampling_topk -1 \
+          -random_sampling_topp 0.95 \
+          -random_sampling_temp 1 \
+          -out /tmp/gen
+        diff data/data_lm/gen-nucleus-sampling-sol.txt /tmp/gen && rm /tmp/gen
+    - name: Test LM generation with random sampling multi-beams
+      run: |
+        python translate.py -model onmt/tests/test_model_lm.pt \
+          -src data/data_lm/src-gen.txt \
+          -verbose -batch_size 10 \
+          -beam_size 10 \
+          -seed 1 \
+          -random_sampling_topk 50 \
+          -random_sampling_topp 0.95 \
```

Comment:
Is it intended to have both `-random_sampling_topk` and `-random_sampling_topp` set here?

Reply:
Yes it is intended, both can be used together